### PR TITLE
Add Dev Containers in VS Code workshop

### PR DESCRIPTION
**Pull request by ornelladotcom**
## Use this template to add a workshop to this repo. Add the following items:

- [x] Your workshop design document, filled in with some details about your proposed workshop including its title and the three stages in the designer delineated, using the `workshop-designer.md` file in the template.

- [x] A `README.md` file that describes the steps of the workshop. 

- [x] Powerpoint Slides for the workshop.

- [ ] The workshop's solution in the `/solution` folder.

- [ ] An embedded Train the Trainer video.

- [x] A Call to Action to enhance the workshop with a direct line to Microsoft Learn by linking to an appropriate Learn module.

A reviewer will be assigned to the workshop and will work with you to shape it for production. Thank you for your contribution!

### CODE REVIEW

File: dev-containers-vscode/workshop-designer.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+# UBD: Understanding By Design
+
+The UBD technique of curriculum design was created by Jay McTighe and Grant Wiggins and outlined in their book, "[Understanding By Design](https://www.amazon.com/Understanding-Design-second-Grant-Wiggins/dp/B006NPF8D0/ref=sr_1_15?crid=26FTBZ6E9Q53K&keywords=understanding+by+design&qid=1636052567&sprefix=understanding+by+design%2Caps%2C94&sr=8-15)". UBD is a process of "backwards design" where the curriculum designer works through three stages, starting with the desired goals of the learning experience and moving backwards through assessments and then to the lesson design. It bears some similarities to TDD (test-driven development) processes in Engineering, where tests are to be written before the code. Use this template to outline your workshop's desired outcomes.
```
Comment (jlooper) [inline dev-containers-vscode/workshop-designer.md:4]: you can delete this

File: dev-containers-vscode/workshop-designer.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+
+# UBD: Understanding By Design
```
Comment (jlooper) [inline dev-containers-vscode/workshop-designer.md:2]: you can delete this

File: dev-containers-vscode/workshop-designer.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Workshop Title
+Getting Started with Development Containers in VS Code
+
+## Workshop Source
+
+[Use a Docker container as a development environment with Visual Studio Code](https://docs.microsoft.com/en-us/learn/modules/use-docker-container-dev-env-vs-code/?WT.mc_id=academic-55190-ornella)
+
+## Stage 1: Desired Results 
+
+> List your established goals of the workshop experience here. These are the takeaways that you want students to have after completing the workshop.
```
Comment (jlooper) [inline dev-containers-vscode/workshop-designer.md:15]: you can delete these callouts

File: dev-containers-vscode/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+# Getting Started with Development Containers in VS Code
+
+## Module Source link
```
Comment (jlooper) [inline dev-containers-vscode/README.md:3]: looking great! Could you comb through for /en-us/ - and remove it, so we don't fix the links to US English

**Review by jlooper — CHANGES_REQUESTED** [review]

File: dev-containers-vscode/README.md (hunk 4 above)
Comment (ornelladotcom) [reply dev-containers-vscode/README.md:3]: done!

**Comment by ornelladotcom** [comment]
@jlooper responded to all feedback, thank you  for reviewing :)

**Review by jlooper — APPROVED** [review]

**Pull request merged**